src/Microsoft.DotNet.Interactive/FormattedValue.cs

```diff
@@ -25,7 +25,17 @@ public FormattedValue(string mimeType, string value)
 
     public string Value { get; }
 
-    public static IReadOnlyList<FormattedValue> FromObject(object value, params string[] mimeTypes)
+    public static FormattedValue CreateSingleFromObject(object value, string mimeType = null)
+    {
+        if (mimeType is null)
+        {
+            mimeType = Formatter.GetPreferredMimeTypesFor(value?.GetType()).First();
```

Comment:
Just thinking out loud: There is an assumption here that the order of returned mime types from `Formatter.GetPreferredMimeTypesFor()' will stay the same. Can this lead to surprising results for external callers of `CreateSingleFromObject()` if we introduce a new mime type in the list (especially if the external caller is calling `CreateSingleFromObject()` indirectly via some other public API  - not sure if this is possible)?

I guess we would not do something like that by accident - but do we have tests to catch if we do?

Reply:
That's definitely expected and is intended to be reconfigured. If you want a specific MIME type, you should pass it. In practice, the reason you don't pass it is because you want whatever defaults the host has configured.